### PR TITLE
#181151229 deploy to Heroku 

### PR DESCRIPTION
**Pull request by semitalatimothy**
### What does this PR do?
 - This PR configures the app to be deployed on Heroku
 
### Description of Task to be completed?
- create heroku account
- configure app for heroku
- create heroku pipeline and review apps
- test app on heroku

### How should this be manually tested?

 -  https://writers-spac-ch-setup-a-nhwemx.herokuapp.com/admin/login/?next=/admin/
 
### Any background context you want to provide?
 -  No
 
### What are the relevant pivotal tracker stories?
 - https://www.pivotaltracker.com/story/show/181151229
 
### Screenshots (if appropriate)

image

### Questions:
 - None
 
[start #181151229]

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['writers-space-invictus-backend.herokuapp.com']
```
Comment (Emmanuel-Dominic) [inline authors/settings.py]: - **why list the hostname like this here?**
- **what is the use of this ALLOWED_HOSTS declaration here?**

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+psycopg2==2.9.3
+python-decouple == 3.6
 PyJWT==2.3.0
 pytz==2021.3
+requests==2.27.1
 six==1.16.0
+sqlparse==0.4.2
 tzdata==2021.5
+urllib3==1.26.8
+whitenoise==6.0.0
```
Comment (Emmanuel-Dominic) [inline requirements.txt]: - **Why all these packages, I think we only need the main packages installed from the terminal?**

**Review by Emmanuel-Dominic — DISMISSED** [review]
**Edit you pull request title and also the project on Heroku is failing with the error below, fix it please as you also ensure the project isn't in DEBUG mode**

heroku-error

File: requirements.txt (hunk 2 above)
Comment (semitalatimothy) [reply requirements.txt]: The packages have been updated.

**Review by Daniel-Kigozi — APPROVED** [review]

**Review by Daniel-Kigozi — APPROVED** [review]

File: authors/settings.py (hunk 1 above)
Comment (semitalatimothy) [reply authors/settings.py]: This has been resolved

**Review by bogerepau — APPROVED** [review]
Looks good welldone Tim

**Review by Victoria129 — APPROVED** [review]

**Review by Emmanuel-Dominic — APPROVED** [review]

**Pull request merged**